src/components/doc/DataBoxSection.vue

```diff
@@ -90,10 +85,110 @@
           {dataBoxId: 'complete', dataBoxType: 'chart', dataBoxSource: 'insee', ...chartData.scatterChart.linked, y: defaultScatterData },
           {dataBoxId: 'complete', dataBoxType: 'chart', dataBoxSource: 'pole-emploi', ...chartData.scatterChart.linked, y: sortedScatterData },
           {dataBoxId: 'complete', dataBoxType: 'chart', dataBoxSource: 'autre', ...chartData.scatterChart.linked, y: reversedScatterData },
-          {dataBoxId: 'complete', dataBoxType: 'table', x: JSON.stringify(JSON.parse(chartData.scatterChart.linked.x)[0]), y: defaultScatterData, name: chartData.scatterChart.linked.name, tableName: 'Années INSEE'},
-          {dataBoxId: 'complete', dataBoxType: 'table', x: JSON.stringify(JSON.parse(chartData.scatterChart.linked.x)[0]), y: reversedScatterData, name: chartData.scatterChart.linked.name, tableName: 'Années'}]"
+          {dataBoxId: 'complete', dataBoxType: 'table', x: tableParse(chartData.scatterChart.linked.x), y: defaultScatterData, name: chartData.scatterChart.linked.name, tableName: 'Années INSEE'},
+          {dataBoxId: 'complete', dataBoxType: 'table', x: tableParse(chartData.scatterChart.linked.x), y: reversedScatterData, name: chartData.scatterChart.linked.name, tableName: 'Années'},
+        ]"
       />
     </div>
+
+    <h3>Un composant flexible pour construire vos tableaux de bord</h3>
+    <p>
+      La databox s’appuie sur
+      <a
+        href="https://www.systeme-de-design.gouv.fr/fondamentaux/grille-et-points-de-rupture/"
+        target="_blank"
+      >
+        la grille proposée par le DSFR
+      </a>
+      , et peut s’afficher selon vos besoins, sur une ou plusieurs colonnes de la grille, grâce à la classe <strong>fr-col</strong>. Pensez toutefois à utiliser un <strong>fr-grid-row</strong> par ligne pour permettre le bon fonctionnement des infobulles du composant.
+    </p>
+    <p>Vous pourrez ainsi construire des dashboard en donnant aux databox la taille nécessaire, en fonction de vos besoins métiers et de l’aspect souhaité pour vos graphiques :</p>
+
+    <div class="fr-mb-6w">
+      <div class="fr-grid-row fr-grid-row--gutters">
+        <div class="fr-col">
+          <data-box
+            v-bind="chartData.dataBox.simple"
+            id="grid-left"
+          />
+          <pie-chart
+            v-bind="chartData.pieChart.doughnut"
+            databox-id="grid-left"
+            databox-type="chart"
+          />
+          <table-chart
+            databox-id="grid-left"
+            databox-type="table"
+            :x="tableParse(chartData.pieChart.doughnut.x)"
+            :y="chartData.pieChart.doughnut.y"
+            name="[&quot;Pourcentage&quot;]"
+            table-name="Catégories"
+          />
+        </div>
+        <div class="fr-col">
```

Comment:
Pour éviter un passage sur plus de 6 colonnes en version tableau : 
```suggestion
        <div class="fr-col-6">
```